server/config/route.js

```diff
@@ -32,6 +33,9 @@ module.exports = function(app, express) {
   app.post('/api/comment/:user_id', postController.addComment);
   app.post('/api/signin/', userController.addUser);
 
+  app.get('/api/guides/', guideController.getAll);
```

Comment:
Do we want the trailing '/' on the routes? I didn't include those on the front-end